cmake/modules/SwiftSharedCMakeConfig.cmake

```diff
@@ -98,7 +98,37 @@ macro(swift_common_standalone_build_config_llvm product)
   endif()
 
   if(LLVM_ENABLE_ZLIB)
-   find_package(ZLIB REQUIRED)
+    find_package(ZLIB REQUIRED)
```

Comment:
Unnecessary whitespace change.

Reply:
It should be two space. I think its pretty harmless of a change.

Reply:
Yeah, it is, just a habitual remark on unnecessary changes :) whitespace changes like that dont actually matter to the functionality